packages/loot-core/src/server/api.ts

```diff
@@ -722,7 +725,7 @@ handlers['api/rule-update'] = withMutation(async function ({ rule }) {
 
 handlers['api/rule-delete'] = withMutation(async function ({ id }) {
   checkFileOpen();
-  return handlers['rule-delete'](id);
+  return handlers['rule-delete']({ id });
```

Comment:
This is a type-only change since the corresponding method in `methods.ts` was updated to match